client/src/Video.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Video = () => {
+  return (
+    <div className="videoWithOverlay">
+      <video className="video" src="https://cdn.pixabay.com/vimeo/636709154/Skyscrapers%20-%2091744.mp4?width=2560&expiry=1657038723&hash=256e8f29b8d7ec5a61fff7e89d7626efb621e350"
```

Comment:
I still cant see any video rendered in the page. Can you please have a look and find why?

Reply:
Video Serving Issue - replaced video with placeholder image for now.